File: smartOrderServer/src/smart/order/server/Command.java
package smart.order.server;

public enum Command {
	
	STOP_CLIENT(0, "CMD_STOP_CLIENT"),
	DEBUG_MSG(1, "CMD_DBG_MSG"),
	
	ACK(11, "CMD_ACK"),
	RECONNECT(12, "CMD_RECONNECT_ON_NEW_PORT");
	
	private final int code;
	private final String cmdTag;
	
	private Command(int code, String cmdTag) {
	    this.code = code;
	    this.cmdTag = cmdTag;
	}
	
	public String cmdTag() {
		return cmdTag;
	}
	
	public int getCode() {
		return code;
	}
}

File: SmartOrderSystem/SmartOrderSystem/Utils/Command.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SmartOrderSystem.Utils
{
  public class Command {
	  
    public static string STOP_CLIENT  = "CMD_STOP_CLIENT";
	  public static string DEBUG_MSG    = "CMD_DBG_MSG";
    public static string ACK          = "CMD_ACK";
	  public static string RECONNECT    = "CMD_RECONNECT_ON_NEW_PORT";
    public static string STILL_ALIVE  = "CMD_STILL_ALIVE";

    public static string EOF = "<EOF>";

  }
}
